docs/src/SUMMARY.md

```diff
@@ -6,5 +6,6 @@
   - [File structure](./description/structure.md)
   - [Type declarations](./description/declarations.md)
   - [Constraint descriptions](./description/constraints.md)
+  - [Ergonomics](./description/ergonomics.md)
```

Comment:
Makes sense. Should it be `Syntactic Sugar` instead of `Convenience Syntax`?

Reply:
personally I don't like the idea of using the phrase "syntactic sugar" in documentation, but we can see what others think. I don't love "convenience syntax", I just haven't thought of anything better. We could use "List comprehension/folding" instead, since it's the only thing that would be in there now, if you prefer that

Reply:
Went with convenience syntax. I'll removed it from this PR though and will move it to another PR.